docs/sections/user_guide/cli/drivers/fv3.rst

```diff
@@ -1,5 +1,5 @@
-Mode ``fv3``
-============
+``fv3``
```

Comment:
I think the "mode" part looks a little odd and unnecessary now that we've split the docs into `Drivers` and `Tools` sections.